roles/kubespray-defaults/tasks/no_proxy.yml

```diff
@@ -26,7 +26,7 @@
 - name: Populates no_proxy to all hosts
   set_fact:
     no_proxy: "{{ hostvars.localhost.no_proxy_prepare }}"
-    proxy_env: "{{ proxy_env | combine({
+    proxy_env: "{{ proxy_env | default({}) | combine({
```

Comment:
Why do we need this change for supporting Ubuntu Focal?

Reply:
# ansible-playbook -i inventory/dev/inventory.yaml reset.yml -u ansible -b
.
.
.
TASK [kubespray-defaults : Populates no_proxy to all hosts] ****************************************************************************************************
fatal: [node001]: FAILED! => {"msg": "|combine expects dictionaries, got AnsibleUndefined"}

Reply:
I guess the above issue happens on Ubuntu 18.04 also, right?
Actually I tired reproducing the above issue on my Ubuntu 20.04 environment, but I cannot reproduce it.
It is better to separate this part as different PR if necessary for concrete scope.

Reply:
maybe related to the ansible version you're using? I agree with @oomichi, create a separate PR for that one if it's really a bug. This PR should be related to focal support only